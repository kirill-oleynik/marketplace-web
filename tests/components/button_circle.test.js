const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ButtonCircle = require('../../app/components/button_circle').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ButtonCircle
        color="grey"
        icon="arrow-up"
        className="test"
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
