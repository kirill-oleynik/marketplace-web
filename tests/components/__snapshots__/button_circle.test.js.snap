// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<button
  className="button-circle button-circle--grey button-circle-- test"
  type="button"
>
  <i
    className="icon icon-arrow-up"
  />
</button>
`;
